source/services/BankidService.js

```diff
@@ -57,6 +57,9 @@ async function auth(ssn) {
     if (response.status === 400) {
       return await auth(ssn);
     }
+    if (response.status !== 200) {
+      throw new Error(response.message);
+    }
```

Comment:
You could simplify this by throw response.message as this is the same as new Error....

Reply:
I did some experimenting and `throw response.message` is not the same as `throw new Error(response.message)` as the latter wraps the message in an actual error object, which for example automatically provides a stack trace. The former would also result in `catch(error){ ... }` catching a `string` instead of an `Error` object which is generally a bad/confusing practice. We should opt to ensure we throw around `Error` objects when possible for consistency.